src/Findologic/Request/Handler/SearchRequestHandler.php

```diff
@@ -77,7 +77,7 @@ public function handleRequest(ShopwareEvent $event): void
      *
      * @throws ServiceNotAliveException
      */
-    public function doRequest(ShopwareEvent $event, ?int $limit = null): Response
+    public function doRequest(ShopwareEvent $event, ?int $limit = null, ?bool $forceFilter = null): Response
```

Comment:
Do we even need `forceFilter` anyway? Seems like it has no usage here after all.

Reply:
It is needed here because it is an abstract method and needs to match the signature of the abstract class. I will check if things work without it, and if so, will remove it altogether.